ui/appui-react/src/appui-react/frontstage/FrontstageDef.tsx

```diff
@@ -316,7 +322,20 @@ export class FrontstageDef {
   }
 
   /** Handles when the Frontstage becomes active */
-  protected _onFrontstageReady(): void { }
+  protected _onFrontstageReady(): void {
+    if (!IModelApp.toolAdmin || !IModelApp.viewManager || !this._initialConfig)
+      return;
+
+    const defaultTool = this._initialConfig.defaultTool;
+    if (!defaultTool) {
+      void IModelApp.toolAdmin.startDefaultTool();
+      return;
+    }
+
+    this._toolAdminDefaultToolId = IModelApp.toolAdmin.defaultToolId;
+    IModelApp.toolAdmin.defaultToolId = defaultTool.toolId;
+    defaultTool.execute();
```

Comment:
Instead of requiring a full tool def, could we only require the tool id ? (Set the Id then startDefaultTool?)

Reply:
Current ToolItemDef allows for custom handlers which support i.e. tool arguments, need to verify if defaultToolId would be enough for now.

Reply:
AFAIK all the defaultTool's we set are registered and could be invoked by just the toolId

I guess you could do a 
```typescript
defaultTool?: ToolItemDef | string;
```
Then you could have it both ways

Reply:
For now I've changed the type to `defaultTool?: string;`. Later we can add a union type w/ tool arguments if needed.

Reply:
The change was not pushed.